sql/core/src/main/scala/org/apache/spark/sql/execution/basicOperators.scala

```diff
@@ -307,6 +307,7 @@ case class MapPartitions[T, U](
     uEncoder: ExpressionEncoder[U],
     output: Seq[Attribute],
     child: SparkPlan) extends UnaryNode {
+  override def missingInput: AttributeSet = AttributeSet.empty
```

Comment:
Yeah, exactly.  I added this when I was debugging a problem with the query planner.